### DISCUSSION TITLE
umi4搭建的项目里面如何集成antd

### DISCUSSION THREAD

**Discussion by Sunny-117** [umijs/umi#10063] [Q&A] [answered]
不用max的情况下，是不是需要手动安装antd

**Comment by Sunny-117** [comment, marked as answer]
antd的css import 'antd/dist/reset.css';  应该加在哪里更合理呢

**Reply by fz6m** [reply]
`src/global.ts` 是全局 js ，可以在这里面加。

https://umijs.org/docs/guides/directory-structure#globaljtsx

**Reply by Sunny-117** [reply]
ok thanks
